db/people.yml

```diff
@@ -3800,10 +3800,21 @@ yztao:
   position: post-doctoral scholar
 zcliu:
   active: true
-  aka: []
+  aka:
+    - Zicheng Liu
+    - Taylor Liu
   avatar: https://github.com/zichengliu1997.png
-  bio: bio
-  education: []
+  bio: Zicheng is from Guangzhou, China. He has spent the last year working as an undergraduate
```

Comment:
Thanks for this @zichengliu1997 !  I see one small typo (rsearch), but more importantly, the github ID should not be in your bio but in a field on its own.  Please search through the db to find an example (search for github)